### ISSUE TITLE
Dynamo 2.0.0  Unable to change costum node name

### ISSUE THREAD

**Issue by fnstegeman** [DynamoDS/Dynamo#8764]
If this issue is with Dynamo for Revit, please post your issue on the [Dynamo for Revit Issues page](https://github.com/DynamoDS/DynamoRevit/issues).

If this issue is **not** a bug report or improvement request, please check the [Dynamo forum](https://forum.dynamobim.com/), and start a thread there to discuss your issue.

## Dynamo version   2.0.0

(Which version of Dynamo are you using? Go to Help > About if you're not sure.)

## Operating system    Windows 7

(e.g. Windows 7, Windows 8.1, etc)

## What did you do? 

Trying to change name costum node

## What did you expect to see?

(Fill in here)

## What did you see instead?

it turn always back in to the original node name

**Comment by fnstegeman** [comment]
![knipsel](https://user-images.githubusercontent.com/23375671/38919349-a0e98064-42f0-11e8-9cef-2c3d4c226a03.PNG)

**Comment by mjkkirschner** [comment]
The name of the custom node in the properties window refers to the name of the function. The name on the node is just a display name for that node instance. If you want to change the actual function name I think you'll need to make a new custom node.

@Racel @ramramps

**Comment by mjkkirschner** [comment]
seems like something we should be able to change in edit custom node properties, but we cannot :(.

**Comment by Racel** [comment]
I don't believe this is a regression. @jnealb @smangarole  can we check now?

I think this is in place to prevent custom node reference failures.
